Echo.json5

```diff
@@ -160,6 +160,7 @@
     { name: 'AREnableAlertDetailsInput', value: true },
     { name: 'AREnableAlertsFilters', value: true },
     { name: 'AREnableAlertsSuggestedFilters', value: true },
+    { name: 'AREnablePartnerOffersNotificationSwitch', value: false },
```

Comment:
Adding a remote configuration for a push notification seems strange to me. There is already a feature flag that prevents partners from initiating partner offers, and we have user-level controls for receiving them, so this would be a third way to control access to this feature.

What's the benefit of adding a remote configuration for this push notification?